src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -89,7 +89,7 @@
  * NOTE: All the public functions in this class also need to be defined in SQLServerConnectionPoolProxy Declare all new
  * custom (non-static) Public APIs in ISQLServerConnection interface such that they can also be implemented by
  * SQLServerConnectionPoolProxy
- * 
+ *
```

Comment:
If possible, please clean up all these whitespace deletions, to reduce the size of the PR.